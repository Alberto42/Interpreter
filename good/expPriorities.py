a = [1]
print(1 + a[0])

print((2 + 2) * 2)
print(2 + 2 * 2)
print(2 * 2 + 2)

print (2 * 2 >= 2 * 1)
print (not False and not False)

print( True or False and False)

