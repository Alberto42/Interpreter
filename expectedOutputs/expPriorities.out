Ident "a" = [1]

2
8
6
6
True
True
True

